regression/pages/common/utils.py

```diff
@@ -0,0 +1,526 @@
+# -*- coding: utf-8 -*-
+
+"""
+Different utilities to be used in tests
+"""
+
+import functools
+from datetime import datetime
+from itertools import izip
+import re
+import time
+import requests
+from requests.auth import HTTPBasicAuth
+
+from edx_rest_api_client.client import EdxRestApiClient
+from guerrillamail import GuerrillaMailSession
+from tempmail import TempMail
+
+from regression.pages.whitelabel.const import (
+    ACCESS_TOKEN,
+    AUTH_PASSWORD,
+    AUTH_USERNAME,
+    URL_WITHOUT_AUTH,
+    DEFAULT_TIMEOUT,
+    ECOMMERCE_URL_WITHOUT_AUTH,
+    ECOMMERCE_API_URL,
+    EMAIL_SENDER_ACCOUNT,
+    INITIAL_WAIT_TIME,
+    ORG,
+    TIME_OUT_LIMIT,
+    WAIT_TIME
+)
+
+
+class MailException(Exception):
+    """
+    Exceptions for Mail failures
+    """
+    pass
+
+
+class ApiException(Exception):
+    """
+    Exceptions for API failures
+    """
+    pass
+
+
+class GuerrillaMailApi(object):
+    """
+    Guerrilla Mail API for creating email accounts and reading emails
+    """
+
+    def __init__(self):
+        self.session = GuerrillaMailSession()
```

Comment:
'session' could get confusing. I suggest you rename this to be more specific, like `email_session`.